lib/snippets.cson

```diff
@@ -5,13 +5,22 @@
       {
         "${1:.source.js}": {
           "${2:Snippet Name}": {
-            "prefix": "${3:hello}",
+            "prefix": "${3:Snippet Trigger}",
             "body": "${4:Hello World!}"
           }
         }
       }$5
     """
 
+  'Atom Snippet Keys':
```

Comment:
Indeed it is...
 
Atom Snippet With No Selector sounds good, but what would be the prefix for that?

Actually, I was thinking about it, what if we do this:

```coffee
'.source.coffee':
  'Atom Snippet':
    prefix: 'snip'
    body: """
      ${1:'.${2:selector}':
        }'${3:Snippet Name}':
          'prefix': '${4:Snippet Trigger}'
          'body': '${5:Hello World!}'$5
    """
```

Instead of creating a new snippet we can make the fist line optional, so type 'snip' hit tab and it will highlight the whole line including the tabs in the next line, if you don't want the selector hit backspace and tab, if you want the selector just hit tab.

What do you think?